docs/team/chuckiex3.md

```diff
@@ -0,0 +1,23 @@
+# Mooi Suet Yeng, Francene's Project Portfolio Page
+
+## Project: Zer0Note
+//todo standardise
+
+### Summary of Contributions
+- Created ASCII art for Zer0Note
+- New Feature: Added the ability to add tasks and notebooks/sections/pages
+- New Feature: Added the ability to select a notebook/section/page
+- Code contributed: [Reposense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=chuckiex3&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=chuckiex3&tabRepo=AY2021S1-CS2113T-T12-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other)
+- Project Management:
+  - Managed release `v2.0` on GitHub
+- Enhancement:
+  - Updated the `select` function so that users can select a notebook/section/page from anywhere [#62](https://github.com/AY2021S1-CS2113T-T12-3/tp/pull/62)
```

Comment:
Move this to after introducing features for flow

Reply:
okay, will do xD